nrf_security/cmake/symbol_rename.cmake

```diff
@@ -0,0 +1,66 @@
+macro(keep_config_test_glue mbedtls_config backend_name)
```

Comment:
Add explanation of the file here like in kconfig_mbedtls_configure.cmake.

Reply:
Will fix